test/index-test.js

```diff
@@ -14,6 +14,7 @@ describe('FontAwesome', () => {
 
   beforeEach(() => {
     const props = {
+      component: 'i',
```

Comment:
@hassankhan could you instead add an additional test for passing in the `component` (or `tagName)` property since it will probably be seldom used by the average user?